lst_scripts/onsite_mc_dl1_to_dl2.py

```diff
@@ -98,7 +98,7 @@ def main(input_dir, path_models=None, config_file=None, flag_full_workflow=False
                 batch_cmd += ' --dependency=afterok:' + wait_ids_proton_and_gammas
             batch_cmd += ' --wrap="{}"'.format(cmd)
 
-            jobid_dl1_to_dl2 = os.popen(batch_cmd).read().split('\n')
```

Comment:
rather than reading the `job_id` from the display, it is set in the variable `SLURM_JOB_ID` by the job manager.

Reply:
but then the variable must be recovered somehow (`echo $SLURM_JOB_ID`) and pass it back to the main code...
I don't see the advantage of this method (to be discussed of course !)

Reply:
The advantage would be to not rely on display (avoiding to catch another message for example)